packages/@okta/vuepress-site/docs/reference/core-okta-api/index.md

```diff
@@ -26,30 +26,321 @@ REST endpoints to configure objects whenever you need. For example:
+| `pr`       | present (has value)   | If the attribute has a non-empty value, or if it contains a non-empty node for complex attributes, there is a match.                                                                                                                                                           |
+| `sw`       | starts with           | The entire operator value must be a substring of the attribute value, starting at the beginning of the attribute value. This criterion is satisfied if the two strings are identical.                                                                                         |
+
+> **Note:** Some objects don't support all the listed operators.
+
+> **Note:** The `ne` (not equal) operator isn't supported for some objects, but you can obtain the same result by using `lt ... or ... gt`. For example, to see all user agents except for "iOS", use `(client.userAgent.os lt "iOS" or client.userAgent.os gt "iOS")`.
+
+> **Note:** All `Date` values use the ISO 8601 format `YYYY-MM-DDTHH:mm:ss.SSSZ`.
+
+#### Attribute operators
+
+| Operator | Description | Behavior                                                         |
+| -------- | ----------- | --------                                                         |
+| `and`      | Logical AND | The filter is only a match if both expressions evaluate to true. |
+| `not` | Logical NOT | The filter is a match if the expression evaluates to false. | 
+| `or`       | Logical OR  | The filter is a match if either expression evaluates to true.    |
+
+#### Logical operators
+
+| Operator | Description         | Behavior                                |
+| -------- | -----------         | --------                                                                                   |
+| `()`       | Precedence grouping | Boolean expressions may be grouped using parentheses to change the standard order of operations: for example, evaluate OR logical operators before logical AND operators. |
+
+Filters must be evaluated using standard order of operations. Attribute operators have the highest precedence, followed by the grouping operator (for example, parentheses), followed by the logical `AND` operator, followed by the logical `OR` operator.
+
+### Hypermedia
+
+Objects in the Okta API use hypermedia for discoverability. Hypermedia enables API clients to navigate objects by following links like a web browser instead of hard-coding URLs in your application. Links are identified by link relations that are named keys. Link relations describe what objects are available and how API clients can interact with them. Each object may publish a set of link relationships based on the state of the object. For example, the status of a user in the [User API](/docs/reference/api/users/#links-object)  governs which lifecycle operations are permitted. Only the permitted operations are published as lifecycle operations.
```

Comment:
```suggestion
Objects in the Okta API use hypermedia for discoverability. Hypermedia enables API clients to navigate objects by following links like a web browser instead of hard-coding URLs in your application. Links are identified by link relations that are named keys. Link relations describe what objects are available and how API clients can interact with them. Each object may publish a set of link relationships based on the state of the object. For example, the status of a user in the [User API](/docs/reference/api/users/#links-object) governs which lifecycle operations are permitted. Only the permitted operations are published as lifecycle operations.
```